docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -1,14 +1,55 @@
 # Development Environment
 
-The development environment setup guide is for setting up your local development environment for JavaScript development; creating plugins and tools for extending WordPress and the block editor.
+This guide is for setting up your local environment for JavaScript development for creating plugins and tools to extend WordPress and the block editor.
 
-A development environment is a catch-all term for the setup on your computer to work. The three main pieces needed for our development environment are:
+A development environment is a catch-all term for what you need setup on your computer to work. The three main pieces needed for our development environment are:
 
 1. Node/NPM Development Tools
 2. WordPress Development Site
 3. Code Editor
 
-## Development Tools
+## Quickstart
+
+Here is a summary of the guide. See each section for additional details and explanations.
+
+**1. Install Node development tools**
+
+Download and install [Node Version Manager](https://github.com/nvm-sh/nvm) (nvm)
+
+```
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+```
+
+Quit and restart terminal
+```
+nvm install --lts
+```
+
+**2. WordPress Development Site**
+
+First download, install, and start [Docker Desktop](https://www.docker.com/products/docker-desktop) following the instructions for your OS.
+
+- Install WordPress environment tool
+
+```
+npm -g install @wordpress/env
+```
+
+Start the environment:
+
+ ```
+ wp-env start
+ ```
+
+You will have a full WordPress site installed, navigate to: http://localhost:8888/ the defualt user is "admin" and default password is "password"
+
```

Comment:
As the above comment is resolved I am adding another comment.

Ahhh....

` wp-env start`

You will have a full WordPress site installed, navigate to: http://localhost:8888/ the defualt user is "admin" and default password is "password"

----> Additional information which is good to know about. 

Another way to open the local WordPress site after it has been started up is through Docker. 
Open Docker and go to the Dashboard. 

![Docker-Dashboard](https://user-images.githubusercontent.com/5323259/92227783-13f51780-eea7-11ea-97fe-7730bf166fe0.png)

Click where it says RUNNING. A new screen opens and it will show the containers (what is a container?). 

![Docker containers](https://user-images.githubusercontent.com/5323259/92227672-e4dea600-eea6-11ea-9a9a-779c0ccfc3b6.png) 

Hovering the cursor over one of the containers shows various options. Among them "OPEN IN BROWSER"
Clicking it will then open the local site in the port mentioned.

Reply:
I think adding the Docker information is confusing

Reply:
What about saying something like this:
"Another option would be to open the browser from Docker by entering the Dashboard, then clicking the stack and selecting a container to open."  Or something like that.

My purpose in adding it is to share information that one can use Docker for various purposes, and this is one of them. Perhaps we can add more Docker information in another place in the docs?

Reply:
We could add it to the more details instructions area?